delivery_carrier_account/models/carrier_account.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2014-TODAY Florian da Costa Akretion <http://www.akretion.com>.
```

Comment:
Please don't put `TODAY`. You have to put `2022` or whatever, but you don't do things in the future.